### ISSUE TITLE
e2e test fails

### ISSUE THREAD

**Issue by iketheadore** [SkycoinProject/skycoin#152]
When running `make test-ui-e2e`, the `ng e2e` test fails with:

```
[11:54:23] I/launcher - Running 1 instances of WebDriver
[11:54:23] I/direct - Using ChromeDriver directly...
[11:54:23] E/launcher - session not created: This version of ChromeDriver only supports Chrome version 76
  (Driver info: chromedriver=76.0.3809.68 (420c9498db8ce8fcd190a954d51297672c1515d5-refs/branch-heads/3809@{#864}),platform=Linux 4.15.0-1028-gcp x86_64)
[11:54:23] E/launcher - SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 76
  (Driver info: chromedriver=76.0.3809.68 (420c9498db8ce8fcd190a954d51297672c1515d5-refs/branch-heads/3809@{#864}),platform=Linux 4.15.0-1028-gcp x86_64)
    at Object.checkLegacyResponse (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/selenium-webdriver/lib/error.js:505:15)
    at parseHttpResponse (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/selenium-webdriver/lib/http.js:509:13)
    at doSend.then.response (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/selenium-webdriver/lib/http.js:440:13)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: WebDriver.createSession()
    at Function.createSession (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/selenium-webdriver/lib/webdriver.js:777:24)
    at Function.createSession (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/selenium-webdriver/chrome.js:709:29)
    at Direct.getNewDriver (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/lib/driverProviders/direct.ts:90:25)
    at Runner.createBrowser (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/lib/runner.ts:225:39)
    at q.then.then (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/lib/runner.ts:391:27)
    at _fulfilled (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/node_modules/q/q.js:796:13)
    at /home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/node_modules/q/q.js:556:49
    at runSingle (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/node_modules/q/q.js:137:13)
    at flush (/home/travis/gopath/src/github.com/SkycoinProject/skycoin/src/gui/static/node_modules/protractor/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
[11:54:23] E/launcher - Process exited with error code 199
npm ERR! code ELIFECYCLE
npm ERR! errno 199
npm ERR! desktopwallet@0.0.0 e2e-choose-config: `ng e2e --proxy-config $E2E_PROXY_CONFIG --delete-output-path false`
npm ERR! Exit status 199
npm ERR! 
npm ERR! Failed at the desktopwallet@0.0.0 e2e-choose-config script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-10-29T11_54_24_059Z-debug.log
+RESULT=199
```

**Comment by iketheadore** [comment]
This error is caused by the mismatched version between webdriver and chrome browser. The webdriver is version `76` and requires the version of chrome browser to be `76` too. From the following logs, we can see that the karam launched the browser of version `78`.

```
2019 11:53:18.868:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:9876/
29 10 2019 11:53:18.870:INFO [launcher]: Launching browser Chrome with unlimited concurrency
29 10 2019 11:53:18.876:INFO [launcher]: Starting browser Chrome
29 10 2019 11:53:41.188:INFO [Chrome 78.0.3904 (Linux 0.0.0)]: Connected on socket o6lzyARggKvUIXmOAAAA with id 19863225
Chrome 78.0.3904 (Linux 0.0.0): Executed 0 of 96 SUCCESS (0 secs / 0 secs)
e 78.0.3904 (Linux 0.0.0): Executed 1 of 96 SUCCESS (0 secs / 0.006 secs)
e 78.0.3904 (Linux 0.0.0): Executed 2 of 96 SUCCESS (0 secs / 0.014 secs)
e 78.0.3904 (Linux 0.0.0): Executed 3 of 96 SUCCESS (0 secs / 0.015 secs)
e 78.0.3904 (Linux 0.0.0): Executed 4 of 96 SUCCESS (0 secs / 0.016 secs)
```

**Comment by iketheadore** [comment]
Fixed in #151
